tests/hpc/ww4.pm

```diff
@@ -74,7 +75,7 @@ sub run ($self) {
         script_run("ssh -o StrictHostKeyChecking=accept-new $node ip a | tee /tmp/script_out");
         $rt = (assert_script_run "grep -E 'inet 192\.168\.10\.11[1-5]' /tmp/script_out", fail_message => 'IP address likely is not set or is not in the defined IP range!!') ? 1 : 0;
         test_case("Check IP on $node", 'Compute Validate IP', $rt);
-        $rt = validate_script_output("ssh -o StrictHostKeyChecking=accept-new $node cat /etc/os-release", sub { m/NAME.+SLE Micro/ });
+        $rt = validate_script_output("ssh -o StrictHostKeyChecking=accept-new $node cat /etc/os-release", sub { m/NAME.+SUSE Linux Enterprise High Performance Computing/ });
```

Comment:
I think would good to also specify container name in variable `HPC_WAREWULF_CONTAINER_NAME` to have  it generic and ready for any container.